README.md

```diff
@@ -23,350 +23,46 @@
-            ["GET", "https://test.k6.io/static/favicon.ico", null, { tags: { staticAsset: "yes" } }],
-            ["GET", "https://test.k6.io/static/js/prisms.js", null, { tags: { staticAsset: "yes" } }],
-        ]);
-        // Combine check() call with failure tracking
-        failureRate.add(!check(resps, {
-            "status is 200": (r) => r[0].status === 200 && r[1].status === 200,
-            "reused connection": (r) => r[0].timings.connecting == 0,
-        }));
-    });
-
-    sleep(Math.random() * 3 + 2); // Random sleep between 2s and 5s
-}
-```
-
-You can save the above example as a local file and run it, or you can also run it directly from the [github copy of the file](https://github.com/grafana/k6/blob/master/samples/thresholds_readme_example.js) with the `k6 run github.com/grafana/k6/samples/thresholds_readme_example.js` command. You can find (and contribute!) more k6 script examples here: [https://github.com/grafana/k6/tree/master/samples](https://github.com/grafana/k6/tree/master/samples)
-
-### Outputs
-
-To make full use of your test results and to be able to fully explore and understand them, k6 can output the raw metrics to an external repository of your choice.
-
-The simplest output option, meant primarily for debugging, is to send the JSON-encoded metrics to a file or to `stdout`. Other output options are sending the metrics to an InfluxDB instance, an Apache Kafka queue, or even to the k6 cloud. This allows you to run your load tests locally or behind a company firewall, early in the development process or as part of a CI suite, while at the same time being able to store their results in the k6 cloud, where you can compare and analyze them. You can find more information about the available outputs [here](https://k6.io/docs/getting-started/results-output) and about k6 Cloud Results [here](https://k6.io/docs/getting-started/results-output/cloud) and [here](https://k6.io/docs/cloud/analyzing-results/overview).
+## Get started
 
-### Modules and JavaScript compatibility
+The docs cover all aspects of using k6. Some highlights include:
+- [Get Started](https://k6.io/docs). Install, run a test, inspect results.
+- [HTTP requests](https://k6.io/docs/using-k6/http-requests/). Have your virtual users use HTTP methods. Or, check the other [Protocols](https://k6.io/docs).
+- [Thresholds](https://k6.io/docs/using-k6/thresholds). Set goals for your test, and codify your SLOs.
+- [Options](https://k6.io/docs/using-k6/k6-options). Configure your load, duration, TLS certificates, and much, much more
+- [Scenarios](https://k6.io/docs/using-k6/scenarios). Choose how to model your workload: open models, closed models, constant RPS, fixed iterations, and more.
+- [Results output](https://k6.io/docs/results-output). Study, filter, and export your test results.
+- [Javascript API](https://k6.io/docs/javascript-api). Reference and examples of all k6 modules.
```

Comment:
```suggestion
- [JavaScript API](https://k6.io/docs/javascript-api). Reference and examples of all k6 modules.
```

I think almost everywhere it has `JavaScript`, so I guess better to keep it :relaxed: